tools/es5to6.js

```diff
@@ -17,91 +16,70 @@ process.on('uncaughtException', handleError);
 const remainingModules = require('./es5to6.json');
 
 const userModules = remainingModules[gitUser()];
+
 if (!userModules || !userModules.length) {
     console.log(chalk.green('⭐️  You have no more modules to convert! ⭐️'));
     process.exit();
 }
 
-git
-    .status((err, status) => {
-        if (
-            status.current !== 'master' ||
-            status.files.length > 0 ||
```

Comment:
without this check, what happens when we switch the user's branch and they have unstaged changes that conflict with master?